mobile/src/pages/Login.tsx

```diff
@@ -0,0 +1,54 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { useState } from 'react';
+import { Button, TextInput, Text, Pressable, View } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  return (
+    <View className={'flex-1 w-full'}>
+      <View className={'w-full flex-row place-content-start mt-16 ml-2'}>
+        <Pressable>
+          <Icon size={45} name="arrow-back-outline" />
+        </Pressable>
+      </View>
+      <View className={"flex-1 flex ml-8 mt-8"}>
```

Comment:
If you run prettier over your code (lmk if you need help with that), it will fix the use of single and double quotes across the file. Consistency is always nice.